app/src/main/res/layout/parking_dialog_fragment.xml

```diff
@@ -12,35 +12,35 @@
         android:layout_width="wrap_content"
         android:layout_height="match_parent"
         android:text="@string/text_button_ok_dialog_fragment"
-        app:layout_constraintTop_toBottomOf="@+id/editTextNumber"
-        app:layout_constraintStart_toStartOf="@+id/textView_dialog_fragment" />
+        app:layout_constraintTop_toBottomOf="@+id/edit_text_number"
+        app:layout_constraintStart_toStartOf="@+id/text_view_dialog_fragment" />
 
     <Button
         android:id="@+id/button_cancel_dialog_fragment"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:text="@string/text_button_cancel_dialog_fragment"
-        app:layout_constraintTop_toBottomOf="@+id/editTextNumber"
-        app:layout_constraintEnd_toEndOf="@+id/textView_dialog_fragment"/>
+        app:layout_constraintTop_toBottomOf="@+id/edit_text_number"
+        app:layout_constraintEnd_toEndOf="@+id/text_view_dialog_fragment"/>
 
     <TextView
-        android:id="@+id/textView_dialog_fragment"
+        android:id="@+id/text_view_dialog_fragment"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:text="@string/text_view_dialog_fragment"
         android:textColor="@color/main_activity_white_text"
-        app:layout_constraintBottom_toTopOf="@+id/editTextNumber"
+        app:layout_constraintBottom_toTopOf="@+id/edit_text_number"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toTopOf="parent" />
 
     <EditText
-        android:id="@+id/editTextNumber"
+        android:id="@+id/edit_text_number"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:ems="5"
```

Comment:
what is em doing?

Reply:
is the size of the font of the number we set for the parking, i put 5 so that the line that is shown in proportion would be short